MAINTAINERS.md

```diff
@@ -0,0 +1,120 @@
+contributors, as maintainers serve mostly as arbiters of if changes are
+acceptable to the contributors to the project in aggregate.
+
+The role of the Maintainer is currently evolving with respect to "Scoped
+Maintainers". The concept of a scoped maintainer is that it is helpful to add
+maintainers who are subject matter experts in a particular area of the code,
+but perhaps not in other areas. As such, maintainers with an explicit scope
+typically will avoid merging PRs in areas outside their scope, but may do so on
+occasion. As an individual maintainer's experience levels in the project
+increase, they may change their focus from scoped to general, but this is not
+currently an explicit designation (all maintainers should be considered to be
+general at this time). As a rough measure, scoped maintainers should avoid
+"stepping on the toes" of work that is being guided by another maintainer or
+contributor.
+
+Historically, one Bitcoin's maintainers was a designated "Lead Maintainer",
+which was most recently @laanwj. However, [in a blog
+post](https://laanwj.github.io/2021/01/21/decentralize.html), @laanwj stated
+that they wanted to take more of a background role as a maintainer and have not
+"passed the baton" to a successor as a Lead Maintainer, and instead focused on
+decentralizing the maintainership ecosystem. As such, while the unofficial
+title of Lead Maintainer still is held by @laanwj, it should be considered a
+bug-not-feature if it confers any additional responsibility or privilege over
+other General Maintainers, and may be explicitly deprecated in the future.
+
+## Self Merges
+
+There is no prohibition or process that differentiates self-merges from normal
+merges.
+
+
+## Nomination and Selection of Maintainers
+
+Being a Maintainer is not an honor (but it may be honorable to serve as one).
+Many regular contributors who are capable or experienced enough choose not to
+be a Maintainer. Similarly, just because a regular contributor has reached a
+certain level of experience does not mean that they "deserve" to be a maintainer.
+
+Maintainers are added by a new maintainer opening a pull-request to add their
+key to the Trusted Keys file. For example, see [this pull
+request](https://github.com/bitcoin/bitcoin/pull/23798). The adding of the key
+is reviewed by the community and merged at the discretion of the existing
+maintainers. Before opening such a pull request, it is typical that the adding
+of the maintainer might be discussed in the regular Bitcoin Core IRC meeting,
+but the actual decision making process occurs in the pull request.
+
+
+## Maintainer Prioritizations
+
+While the maintainers are ultimately not accountable to any particular roadmap,
+nor can make regular contributors focus on one area or another, practically the
+views and prioritizations of maintainers have an impact on what gets reviewed,
+merged, and the general direction of the project.
```

Comment:
Removing it would seem to me be prescriptive (aspirational), rather than descriptive of the current status quo. We should aim to debate prescriptive additions or omissions in follow up work?